Examples/AblyPush/AblyPushExample/ContentView.swift

```diff
@@ -62,11 +91,78 @@ struct ContentView: View {
                     }
                     return Alert(title: Text("Device Details Error"), message: Text("Unknown result."))
                 }
-                .padding()
-                Button("Send Push") {
-                    AblyHelper.shared.sendAdminPush(title: "Hello", body: "This push was sent with deviceId")
+                Text("Device Push")
+                    .fontWeight(.bold)
+                    .padding(.top)
+                StatelessButton(
+                    isButtonEnabled: ablyHelper.isPushActivated,
+                    systemImage: "iphone.gen3",
+                    title: "Send Push to deviceId",
+                    action: {
+                        AblyHelper.shared.sendAdminPush(title: "Hello", body: "This push was sent with deviceId")
+                    }
+                )
+                .fixedSize(horizontal: false, vertical: true)
+                Text("Channels Push")
+                    .fontWeight(.bold)
+                    .padding(.top)
+
+                HStack {
+                    PushButton(
+                        isButtonEnabled: ablyHelper.isPushActivated,
+                        systemImage: ablyHelper.isSubscribedToExampleChannel1
+                        ? "checkmark.circle.fill"
```

Comment:
Should be "xmark.circle.fill" in case of subscribe.

Reply:
Usually we put a comment with the commit link under the thread and the one who asked should resolve.

https://github.com/ably/ably-cocoa/pull/1879/commits/6a68d68f43de37cc0285fd4881e2910ba1818c39